app/main.py

```diff
@@ -1,2 +1,58 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self,
+                 capacity: int = 8,
+                 load_factor: float = 2 / 3,
+                 size: int = 0) -> None:
+        self.capacity = capacity
+        self.load_factor = load_factor
+        self.size = size
+        self.table = [None] * capacity
+
+    def __getitem__(self, key: Hashable) -> Any:
+        index = self.find_index(key)
+        if self.table[index] is None:
+            raise KeyError(f"Key {key} does not found")
+        return self.table[index][1]
+
+    def __setitem__(self, key: Hashable, value: Any) -> None:
+        if not self.size < self.capacity * self.load_factor:
+            self.resize()
+        index = self.find_index(key)
+        if self.table[index] is None:
+            self.size += 1
+        self.table[index] = [key, value, hash(key)]
+
+    def __len__(self) -> int:
+        return self.size
+
+    def __delitem__(self, key: Hashable) -> None:
+        index_key = self.hash_func(key)
+        if self.table[index_key] and self.table[index_key][0] == key:
+            del self.table[index_key]
+            self.size -= 1
+
+    def clear(self) -> None:
+        self.table = []
+        self.size = 0
```

Comment:
it seems to me, that u should do the same thing as in init method
make capacity equal to 8 and then make table with 8 spaces in it )

Reply:
u didn't fix that, i mean clear method, that should start from capacity = 8, and table [None] * capacity )